tools/spin-ec2/main.go

```diff
@@ -26,9 +31,24 @@ const (
 	colorWhite  = "\033[37m"
 )
 
-var letterRunes = []rune("abcdefghijklmnopqrstuvwxyz")
+var (
+	letterRunes = []rune("abcdefghijklmnopqrstuvwxyz")
+	hostPrefix  = "canary"
+)
 
 func main() {
+	interactive := flag.Bool("i", false, "run the CLI in interactive mode")
```

Comment:
what if we do interactive by default? For script we do it once and forget, but for interactive we do it each time again and will have to add `-i=true`